docs/source/features/screening.rst

```diff
@@ -119,11 +119,13 @@ Download Results
 A file containing all metadata including your decissions can be downloaded
 any time during the screening process. To download your results:
 
-1. Open ASReview LAB.
-2. Open a project.
-3. Start screening.
-4. Click the **download** icon in the upper-right corner.
-5. You will be asked whether you want to save an Excel or a CSV file.
+1. Open ASReview LAB. 
+2. Open a project. 
+3. Start screening. 
+4. Click the
```

Comment:
This line renders an error in the documentation: 

`Enumerated list ends without a blank line; unexpected unindent.`